src/Smalot/PdfParser/Font.php

```diff
@@ -391,20 +394,18 @@ public function decodeText($commands)
         }
 
         foreach ($words as &$word) {
-            $loop_unicode = $unicode;
-            $word = $this->decodeContent($word, $loop_unicode);
+            $word = $this->decodeContent($word);
         }
 
         return implode(' ', $words);
     }
 
     /**
      * @param string $text
-     * @param bool   $unicode
      *
      * @return string
      */
-    public function decodeContent($text, &$unicode)
+    public function decodeContent($text)
```

Comment:
You changed the function head of a public function which is part of the API. It requires us to bump major version. Can we avoid that?

Reply:
I could leave the parameter in for now, so it just wouldn't be used anymore. What would be the best way to mark it as deprecated?

Reply:
I suggest we use the follow message here: `@deprecated Usage of second parameter $unicode is deprecated. It might be removed in a future release.`

I also noticed that without `$unicode` being used inside, the behavior of this function changes. Is that right? That might also be a blocker and requires the bump of the major version.

Reply:
With the changed code, the `$unicode` variable is no longer used from the arguments. There still is a $unicode variable in the function body, but it's always set with a value returned by `mb_check_encoding()`.

Reply:
The `@deprecated` marks the whole method as deprecated (at least in VS Code with Inteliphense).
![grafik](https://user-images.githubusercontent.com/13076806/94996191-59d6f580-05a3-11eb-81bc-16c059aa314d.png)

Maybe it would be better to add a description to the parameter itself? This looks fine:
![grafik](https://user-images.githubusercontent.com/13076806/94996205-6d825c00-05a3-11eb-9f8f-30da3d1d5129.png)

Reply:
You are right, good point.

Reply:
Done!